Source/Supporting Files/Utilities.swift

```diff
@@ -117,7 +117,7 @@ extension DispatchTimeInterval {
 
 extension DispatchQueue {
```

Comment:
@hartbit Does this part look good to you as well, this is the only place I was a bit uncertain.